### PR TITLE
DOC: update the Series.isin docstring

### PR DESCRIPTION
**Pull request by DaanVanHauwermeiren**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [x] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [x] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [ ] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
######################## Docstring (pandas.Series.isin) ########################
################################################################################

Check whether ``values`` are contained in Series.

Return a boolean Series showing whether each element in the Series
is exactly contained in the passed sequence of ``values``.

Parameters
----------
values : set or list-like
    The sequence of values to test. Passing in a single string will
    raise a ``TypeError``. Instead, turn a single string into a
    ``list`` of one element.

    .. versionadded:: 0.18.1

    Support for values as a set.

Returns
-------
isin : Series (bool dtype)

Raises
------
TypeError
  * If ``values`` is a string

See Also
--------
pandas.DataFrame.isin : equivalent method on DataFrame

Examples
--------

>>> s = pd.Series(['lama', 'cow', 'lama', 'beetle', 'lama',
...               'hippo'], name='animal')
>>> s.isin(['cow', 'lama'])
0     True
1     True
2     True
3    False
4     True
5    False
Name: animal, dtype: bool

Passing a single string as ``s.isin('lama')`` will raise an error. Use
a list of one element instead:

>>> s.isin(['lama'])
0     True
1    False
2     True
3    False
4     True
5    False
Name: animal, dtype: bool

################################################################################
################################## Validation ##################################
################################################################################

Docstring for "pandas.Series.isin" correct. :)
```

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -2771,27 +2772,33 @@ def isin(self, values):
...
-        pandas.DataFrame.isin
+        pandas.DataFrame.isin : equivalent method on DataFrame
 
         Examples
         --------
 
-        >>> s = pd.Series(list('abc'))
-        >>> s.isin(['a', 'c', 'e'])
+        >>> s = pd.Series(['lama', 'cow', 'lama', 'beetle', 'lama',
+        ...               'hippo'], name='animal')
```
Comment (TomAugspurger) [inline pandas/core/series.py]: PEP8: add one more space before `'hippo'`

File: pandas/core/series.py [hunk 2]
```diff
@@ -2745,9 +2745,10 @@ def _take(self, indices, axis=0, convert=True, is_copy=False):
 
     def isin(self, values):
         """
-        Return a boolean :class:`~pandas.Series` showing whether each element
-        in the :class:`~pandas.Series` is exactly contained in the passed
-        sequence of ``values``.
+        Check whether ``values`` are contained in ``Series``.
+
+        Return a boolean ``Series`` showing whether each element in the
+        ``Series`` is exactly contained in the passed sequence of ``values``.
```
Comment (kynan) [inline pandas/core/series.py]: consider wording as "matches an element in the passed sequence of ``values`` exactly"

**Review by kynan — APPROVED** [review]

**Comment by TomAugspurger** [comment]
Minor comment. LGTM otherwise.

File: pandas/core/series.py (hunk 1 above)
Comment (DaanVanHauwermeiren) [reply pandas/core/series.py]: updated in 515aecc

File: pandas/core/series.py (hunk 2 above)
Comment (DaanVanHauwermeiren) [reply pandas/core/series.py]: updated in c17e423

**Review by jorisvandenbossche — APPROVED** [review]

**Pull request merged**